rust/exchange_rate/src/main.rs

```diff
@@ -40,6 +41,11 @@ pub const DATA_POINTS_PER_API: u64 = 200;
 // 10 (bytes per field) * 6 (fields per timestamp) * 200 (timestamps)
 pub const MAX_RESPONSE_BYTES: u64 = 10 * 6 * DATA_POINTS_PER_API;
 
+#[derive(Serialize, Deserialize)]
+struct Context {
+    indexes: Vec<u8>,
```

Comment:
sorry, I was not clear enough I meant something different

I think here the fields should be 

```
bucket_start_time_index: usize
closing_price_index: usize

```